sdks/python/container/boot.go

```diff
@@ -33,6 +34,7 @@ import (
 	"github.com/apache/beam/sdks/go/pkg/beam/util/execx"
 	"github.com/apache/beam/sdks/go/pkg/beam/util/grpcx"
 	"github.com/golang/protobuf/proto"
+	"github.com/nightlyone/lockfile"
```

Comment:
For my learning, how does one pick/vet right go packages?

Reply:
About the same way as any other package: looking at the code. There's no good third party vetting system for Go at the moment.

As for this one, it's a small package, (~200 lines), and only imports from the standard library, making it relatively safe, and easy to inspect.
https://github.com/nightlyone/lockfile/blob/master/lockfile.go